travelrec/scoring.py

```diff
@@ -15,27 +15,29 @@ def score_city(city, params):
             city_latitude, city_longitude, search_distances['geofeatures'], geofeatures_codes
         )
         geofeatures_res = geo_results["results"]["bindings"]
-        geofeatures_count = len(geofeatures_res)
         geofeatures_places = [{
             'name': gf['name']['value'],
             'coordinates': f"POINT({gf['long']['value']} {gf['lat']['value']})",
-            'distance': gf['distance_km']['value']
-            } for gf in geofeatures_res] 
+            'distance': gf['distance_km']['value'],
+            'score': (search_distances['geofeatures'] - float(gf['distance_km']['value'])) / search_distances['geofeatures']
```

Comment:
Extract this to function to have score calculation in one place. Something like:
```
def calculate_score(search_radius: float, distance: float):
    return (search_radius - distance)/search_radius
```
Add code documentation and UI hint to indicate that score is normalized i.e. in [0, 1] range.